cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-analytics.js

```diff
@@ -188,7 +184,7 @@ const handlePaginationClick = (event) => {
  * @returns {HTMLElement|null} The checkbox div or null if it's not a checkbox
  */
 const getClearBtn = (event) => {
-  const el = closest(event.target, '.results_filters-clear') || event.target;
+  const el = event.target.closest('.results_filters-clear') || event.target;
```

Comment:
TRIVIAL: Similar to earlier, you could pull `event.target` into its own constant to avoid the possible double-lookup.